pkg/openstack/loadbalancer.go

```diff
@@ -1003,6 +1004,39 @@ func (lbaas *LbaasV2) buildBatchUpdateMemberOpts(port corev1.ServicePort, nodes
 	return members, newMembers, nil
 }
 
+func derefOrEmpty[T any](p *T) T {
+	if p != nil {
+		return *p
+	}
+	var t T
+	return t
+}
```

Comment:
it makes sense to define this func in `pkg/util`

Reply:
Good point. I have replaced that with a call to `ptr.Deref` in k8s.io/utils

Reply:
Even better. Thanks!